src/MessageHandler/ActivityPub/Inbox/CreateHandler.php

```diff
@@ -40,23 +42,20 @@ public function __construct(
     public function __invoke(CreateMessage $message): void
     {
         $this->object = $message->payload;
-        $this->logger->debug('Got a CreateMessage of type {t}', [$message->payload['type'], $message->payload]);
-
+        $this->logger->debug('Got a CreateMessage of type {t}, {m}', ['t' => $message->payload['type'], 'm' => $message->payload]);
         try {
-            if ('Note' === $this->object['type']) {
+            if (!$this->activityPubManager->isActivityPublic($message->payload)) {
```

Comment:
Do you think this is the best way of checking for a private message rather than checking on type `ChatMessage`? 😕

Reply:
No its not. I should definitely just check the type. I'll change this. The original thought was the private mentions from mastodon. So I will not delete the medthod declaration, since it will be useful later on when implementing that

Reply:
done